featured-mods/funkyfs/meta.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "Funky Friend Server",
     "description": "A mod about a ...bunch of friends???? ",
-    "author": "Tobi_7377",
```

Comment:
Please verify that this author change has authorization

Reply:
![Screenshot_2024-12-26-16-16-09-05_572064f74bd5f9fa804b05334aa4f912](https://github.com/user-attachments/assets/d6ccc5ec-fdb8-47ba-8899-03d691ed050b)
![Screenshot_2024-12-26-16-15-45-94_572064f74bd5f9fa804b05334aa4f912](https://github.com/user-attachments/assets/84d86fdd-52b7-4461-bb67-d644763fed6f)
![Screenshot_2024-12-26-16-15-31-72_572064f74bd5f9fa804b05334aa4f912](https://github.com/user-attachments/assets/0030f874-2852-450d-a8c8-2fde739d366c)

mb i typed his youtube name before😭